controllers/volumereplicationgroup_controller.go

```diff
@@ -276,7 +277,7 @@ func filterPVC(mgr manager.Manager, pvc *corev1.PersistentVolumeClaim, log logr.
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.7.0/pkg/reconcile
 func (r *VolumeReplicationGroupReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	log := r.Log.WithValues("VolumeReplicationGroup", req.NamespacedName)
+	log := r.Log.WithValues("VolumeReplicationGroup", req.NamespacedName, "Rid", uuid.New())
```

Comment:
Why we don't use .WithName("controllers").WithName("vrg") for this logger?

It will be nice to have one function for configuring the logger that we can use in all controllers so all logs have the same format.